src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_LENGTH = 10;
+    private Entry<K, V>[] items;
+    private int length = 0;
+
+    public StorageImpl() {
+        this.items = new Entry[MAX_LENGTH];
+    }
+
     @Override
     public void put(K key, V value) {
+        Entry<K, V> entry = new Entry<>(key, value);
+        for (int i = 0; i < length; i++) {
+            if ((items[i].getKey() == null && key == null)
+                    || (items[i].getKey() != null && items[i].getKey().equals(key))) {
+                items[i] = entry; // если одинаковые объекты - обновляем(перезаписываем)
```

Comment:
```suggestion
                items[i] = entry;
```
коментарі в коді доцільні коли у нас приймається якесь неочевидне рішення, щоб пояснити чому код так повинен працювати, в інших випадка код повинен бути зрозумілий без коментарів